{"name": "Test generate token with invalid accounts", "status": "passed", "parameters": [{"name": "account", "value": "<data_object.account.Account object at 0x000002459C290260>"}], "start": 1702281248690, "stop": 1702281249843, "uuid": "0513575a-a481-4b28-8cb4-8acc4c3268e0", "historyId": "9921a1e434679bd8e4846f003b68cabf", "testCaseId": "573dcf4d17f28e829fe2a39fec07d16d", "fullName": "testcases.api.test_generate_token#test_generate_token_unsuccessfully_with_invalid_account", "labels": [{"name": "parentSuite", "value": "API Test"}, {"name": "suite", "value": "Test Generate Token"}, {"name": "subSuite", "value": "TestGenerateToken"}, {"name": "host", "value": "VNNOT02222"}, {"name": "thread", "value": "13036-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api.test_generate_token"}]}